pkg/database/rql/Database.go

```diff
@@ -0,0 +1,210 @@
+package rql
+
+import (
+	"fmt"
+	"github.com/quan-to/chevron/internal/config"
+	"github.com/quan-to/chevron/internal/tools"
+	r "gopkg.in/rethinkdb/rethinkdb-go.v6"
+	"time"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports).